app/src/main/java/org/oppia/android/app/customview/VerticalDashedLineView.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.android.app.customview
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.DashPathEffect
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.util.TypedValue
+import android.view.View
+import androidx.core.content.ContextCompat
+import org.oppia.android.R
+
+private const val DASH_GAP_IN_DP = 16
+private const val DASH_LENGTH_IN_DP = 32
+private const val DASH_THICKNESS_IN_DP = 6
+
+/**
+ * Custom view to represent chapter progress in [StoryActivity].
```

Comment:
This doc string is a bit too tied to how it's used, I think. Consider instead: ``Custom view for drawing a vertical dashed line.``

It might also be nice to specify how to customize it & properly add it to a view (i.e. if its width/height need to be specified in a particular way).

Reply:
Updated as below:
```
/**
 * Custom view for drawing a vertical dashed line.
 * This view can be customised by changing the values of [DASH_GAP_IN_DP], [DASH_LENGTH_IN_DP] and
 * [DASH_THICKNESS_IN_DP].
 * Reference: https://stackoverflow.com/a/27054463
 */
```